config-base.js

```diff
@@ -108,19 +119,35 @@ module.exports = {
     'unicorn/catch-error-name': 'off',
     'unicorn/consistent-function-scoping': 'off',
     'unicorn/no-abusive-eslint-disable': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-await-expression-member': 'off',
     'unicorn/no-fn-reference-in-iterator': 'off',
     'unicorn/no-nested-ternary': 'warn',
     'unicorn/no-null': 'off',
     'unicorn/no-process-exit': 'off',
     'unicorn/no-reduce': 'warn',
+    'unicorn/numeric-separators-style': 'off',
+    'unicorn/prefer-includes': 'off',
```

Comment:
we commonly use `array.some` when using enum values. Using `includes` in those cases results in a type error that warrants either a typeguard, casting, or an eslint ignore so I thought it'd be better to just turn this rule off

Reply:
That makes sense